completions/cryptsetup

```diff
@@ -40,10 +40,14 @@ _cryptsetup()
             COMPREPLY=($(compgen -W '$(_parse_help "$1")' -- "$cur"))
             [[ ${COMPREPLY-} == *= ]] && compopt -o nospace
         else
-            COMPREPLY=($(compgen -W 'open close resize status benchmark
-                repair erase luksFormat luksAddKey luksRemoveKey luksChangeKey
-                luksKillSlot luksUUID isLuks luksDump tcryptDump luksSuspend
-                luksResume luksHeaderBackup luksHeaderRestore' -- "$cur"))
+            COMPREPLY=($(compgen -W 'benchmark bitlkClose bitlkDump bitlkOpen
```

Comment:
If we want to keep hardcoding the list, could we add some pointers here to the source of the list, as well as how to keep the list sorted for easy maintenance? (I believe it before this change the entries were roughly in the order the man page has them, which wasn't noted either, though, but we're here to improve on things ;))

But actually, I think we should consider switching to scraping `--help` output instead of hardcoding. For example on Ubuntu 20.04, it contains
```
<action> is one of:
	open <device> [--type <type>] [<name>] - open device as <name>
	close <name> - close device (remove mapping)
	resize <name> - resize active device
	status <name> - show device status
	benchmark [--cipher <cipher>] - benchmark cipher
	repair <device> - try to repair on-disk metadata
	reencrypt <device> - reencrypt LUKS2 device
	erase <device> - erase all keyslots (remove encryption key)
	convert <device> - convert LUKS from/to LUKS2 format
	config <device> - set permanent configuration options for LUKS2
	luksFormat <device> [<new key file>] - formats a LUKS device
	luksAddKey <device> [<new key file>] - add key to LUKS device
	luksRemoveKey <device> [<key file>] - removes supplied key or key file from LUKS device
	luksChangeKey <device> [<key file>] - changes supplied key or key file of LUKS device
	luksConvertKey <device> [<key file>] - converts a key to new pbkdf parameters
	luksKillSlot <device> <key slot> - wipes key with number <key slot> from LUKS device
	luksUUID <device> - print UUID of LUKS device
	isLuks <device> - tests <device> for LUKS partition header
	luksDump <device> - dump LUKS partition information
	tcryptDump <device> - dump TCRYPT device information
	luksSuspend <device> - Suspend LUKS device and wipe key (all IOs are frozen)
	luksResume <device> - Resume suspended LUKS device
	luksHeaderBackup <device> - Backup LUKS device header and keyslots
	luksHeaderRestore <device> - Restore LUKS device header and keyslots
	token <add|remove|import|export> <device> - Manipulate LUKS2 tokens

You can also use old <action> syntax aliases:
	open: create (plainOpen), luksOpen, loopaesOpen, tcryptOpen
	close: remove (plainClose), luksClose, loopaesClose, tcryptClose
```
CentOS 7's `cryptsetup --help` emits similar looking sections, so I suppose this would be feasible to do in at least somewhat portable manner.

Reply:
Thank you for the comment! I have updated the PR to include the analysis of `--help` (790e524). What is non-trivial is that not all the actions are found in the output of `--help`. For example, for `cryptsetup` 2.4.3 in Fedora 36, `--help` does not contain the reported `luksErase` (#758) while `man` contains the description of `luksErase`. I have decided to also analyze the `man` output.

Reply:
> For example, for `cryptsetup` 2.4.3 in Fedora 36, `--help` does not contain the reported `luksErase`

Perhaps upstream would appreciate a bug report about this?